core/new-gui/src/app/workspace/service/dynamic-schema/auto-attribute-correction/auto-attribute-correction.service.ts

```diff
@@ -24,9 +22,7 @@ import { environment } from "src/environments/environment";
  * - When user deletes an attribute (e.g., deselected in a projection operator),
  * all succeeding operators containing the attribute will delete the attribute from themselves and become invalid.
  */
-@Injectable({
-  providedIn: "root",
-})
+@Injectable()
```

Comment:
This line change would cause the following error on my end.
```
Uncaught (in promise): NullInjectorError: R3InjectorError(AppModule)[AutoAttributeCorrectionService -> AutoAttributeCorrectionService -> AutoAttributeCorrectionService]: NullInjectorError: No provider for AutoAttributeCorrectionService!
```

Changing it back to
```
@Injectable({
  providedIn: "root",
})
```
fixes the problem.